gns3/modules/qemu/dialogs/qemu_vm_wizard.py

```diff
@@ -285,6 +289,13 @@ def getSettings(self):
             settings["default_symbol"] = ":/symbols/iosv_l2_virl.normal.svg"
             settings["hover_symbol"] = ":/symbols/iosv_l2_virl.selected.svg"
             settings["category"] = Node.switches
+        elif self.uiTypeComboBox.currentText() == "IOS-XRv":
+            settings["adapters"] = 16
+            settings["hda_disk_image"] = self.uiHdaDiskImageLineEdit.text()
+            settings["default_symbol"] = ":/symbols/router.normal.svg"
+            settings["hover_symbol"] = ":/symbols/router.selected.svg"
+            settings["options"] = "-enable-kvm"
```

Comment:
Oh, you mean if IOS-XRv works only on Linux? No, it does not matter, it is a virtual machine just like any other QEMU machine.